sql/setup.sql

```diff
@@ -1,23 +1,6 @@
 CREATE DATABASE db_PICASO_OdhranH;
 USE db_PICASO_OdhranH;
 
-CREATE TABLE Client (
-  ClientID int AUTO_INCREMENT PRIMARY KEY,
-    Forename varchar(20) NOT NULL,
-    Surname varchar(20) NOT NULL,
-    Address varchar(50) NOT NULL,
-    Phone varchar(15) NOT NULL
-);
-
-# SQL for creating Delivery Employee table
-CREATE TABLE DeliveryEmployee (
-	DEmpID INT AUTO_INCREMENT NOT NULL PRIMARY KEY,
-	DEmpForename VARCHAR(20) NOT NULL,
-	DEmpSurname VARCHAR(20) NOT NULL,
-	DEMPBankNum VARCHAR(16) NOT NULL,
-	NationalInsuranceNum VARCHAR(9) NOT NULL
-);
-
 CREATE TABLE Project (
   ProjectID int NOT NULL AUTO_INCREMENT,
```

Comment:
Your tabs are inconsistent with the rest of the file. Not blocking the merge but please fix this in future commits! :)